cmd/src/lsif_upload_flags.go

```diff
@@ -155,35 +156,43 @@ type argumentInferenceError struct {
 	err      error
 }
 
-func handleLSIFTyped(out *output.Output) error {
-	if strings.HasSuffix(lsifUploadFlags.file, ".lsif-typed") {
-		// The user explicitly passed in a -file flag that points to an LSIF Typed index.
+func replaceExtension(oldPath string, newExtension string) string {
+	return oldPath[:len(oldPath)-len(path.Ext(oldPath))] + newExtension
```

Comment:
```suggestion
	return filepath.Join(filepath.Dir(oldPath), filepath.Base(oldPath) + newExtension)
```

Reply:
I don't think that is correct; it will only attach the extension without removing the old extension.

Reply:
Oh no I miscomprehended the entire stdlib 😓